package.json

```diff
@@ -49,7 +49,7 @@
     "regenerator-runtime": "^0.13.3"
   },
   "dependencies": {
-    "isomorphic-fetch": "^2.2.1",
```

Comment:
does it mean we'll use default browser `fetch`? or is there an override somewhere else?

Reply:
ah, I see, `isomorphic-fetch` doesn't support signals

Reply:
We should get rid of isomorphic-fetch anywhere we see it provided it doesn't break the tests. That was a holdover from like 2015 when all browsers we considered plausibly supporting didn't necessarily support fetch and we were still considering using node based testing at least some of the time. And we still do in connect! However, I'm pretty sure its use of fetch-mock should mean it doesn't matter if we ditch isomorphic.